Dockerfile

```diff
@@ -110,6 +112,28 @@ RUN dnf install -y \
     wget \
   && dnf clean all
 
+# Install Microsoft Edge & webdriver
+RUN rpm --import https://packages.microsoft.com/keys/microsoft.asc \
+  && dnf config-manager --add-repo https://packages.microsoft.com/yumrepos/edge \
+  && dnf makecache \
+  && dnf install -y \
+    microsoft-edge-stable-${EDGE_VERSION} \
+    wget \
+    zip \
+
+  && wget -q "https://msedgedriver.azureedge.net/${EDGE_VERSION}/edgedriver_linux64.zip" \
+  && unzip edgedriver_linux64.zip -d edge \
+  && mv edge/msedgedriver /opt/robotframework/drivers/ \
+  && rm -Rf edge/ \
+  && rm -f edgedriver_linux64.zip \
+
+  && dnf remove -y \
+    zip \
+  && dnf clean all
+
+# WARNING:
+# don't remove the wget package after this point because it's a dependency of Microsoft Edge
+
```

Comment:
Thank you for this explanation, I moved it a bit higher up the file.
```suggestion
```

Reply:
changed